data/transport missions.txt

```diff
@@ -682,6 +690,7 @@ mission "There Might Be Riots 1"
 	on complete
 		payment
 		payment 100000
+		log `Helped to transport a band known as "There Might Be Riots" to their concert on Wayfarer. Their music is bizzare, but they pay well.`
```

Comment:
> Helped to transport a band...
I feel 'to' is redundant here, and it can just be written as 'Helped transport a band...' ?